homeassistant/components/openweathermap/config_flow.py

```diff
@@ -84,6 +87,9 @@ async def async_step_user(self, user_input=None):
                 vol.Optional(CONF_LANGUAGE, default=DEFAULT_LANGUAGE): vol.In(
                     LANGUAGES
                 ),
+                vol.Optional(
+                    CONF_SCAN_INTERVAL, default=DEFAULT_WEATHER_UPDATE_INTERVAL
```

Comment:
We don't allow a scan interval option. The user can turn off automatic updates on all integrations that support config entries. Then the user can automate updates at the interval they want, with the entity service `homeassistant.update_entity`.

Reply:
That just seems silly to me that a user would need to create an automation to update something more frequently when it can be easily added as an optional configuration instead. Just does not seem "user friendly" to me.

Reply:
Sure, we get your point of view; however, we also have been flooded in the past with requests on other variants. For example: Slower interval during the night, a higher interval during the day, no polling when not home, or more polling during the weekend and many other variants/ideas.

The `homeassistant.update_entity` was introduced to provide a single way for everybody to create whatever they want or need.

../Frenck